lab-steven/test/test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const assert = require('assert');
+
+describe('Greet', function() {
+  describe('#greet', function() {
+    it('Expected to output "hello <name>"', function() {
+      var result = greet.greet(process.argv[2]);
+      assert.ok(result === `hello ${process.argv[2]}`, `Instead says ${result}`);
+    });
+
+    it('Name expected to be undefined when using mocha', function() {
+      assert.ok(greet.greet() === 'hello undefined', 'Did not come back as hello undefined');
+    });
+  });
+});
```

Comment:
Stoked to see you using process.argv[2] to solve the bonus! Good job. As a minor nitpick, it would be considered best practice to abstract your method that dynamically takes in a name parameter as a command line argument into its own method. In that case, then you could keep your module.exports = exports = { } that you declared originally in greet.js and just attach another method onto the exports object.